common/views/components/ImageViewer/ImageViewer.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import {font, spacing} from '../../../utils/classnames';
+import {convertImageUri, convertIiifUriToInfoUri} from '../../../utils/convert-image-uri';
+import ButtonButton from '../Buttons/ButtonButton/ButtonButton';
+
+const buttonFontClasses = font({s: 'HNM5'});
+const commonBtnTracking = (id, trackTitle) => {
+  return `"category": "component", "label": "id:${id}', title:${trackTitle}"`;
+};
```

Comment:
Why not whack this in the button component?

Reply:
Good question – I'll take a look.

Reply:
I'm guessing this tracking is common to the three buttons that are in the `ImageViewer` rather than common to buttons in general. Think it should stay here.